gen/calc.py

```diff
@@ -86,6 +86,22 @@ def calculate_mesos_slave_modules_json(mesos_slave_modules):
     return json_multiline.replace('\n', '\n' + injected_indent)
 
 
+def calculate_mesos_slave_modules(dcos_overlay_enable):
```

Comment:
This really breaks adding more downstream modules... Would it be reasonable to wait to land this until the modules dir stuff @karya0 has been working on lands? That makes this really simple to do.